src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java

```diff
@@ -90,16 +93,19 @@
       + "configurations to start polling for data in added tables or stop polling for data in "
       + "removed tables.";
   public static final long TABLE_POLL_INTERVAL_MS_DEFAULT = 60 * 1000;
+  private static final String TABLE_POLL_INTERVAL_MS_DISPLAY = "Table Poll Interval (ms)";
```

Comment:
Data dictionary monitor interval (ms), or metadata monitor interval (ms). We are not actually polling a table here...